src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -1,4 +1,5 @@
 import { AfterContentInit, Component, ElementRef, OnInit, ViewChild } from "@angular/core";
+import { DomSanitizer, SafeHtml } from "@angular/platform-browser";
```

Comment:
It doesn't make sense to duplicate this code in all components. Instead the base component could expose a `valueHTML` or similar property that can be used in components where required